src/assets/icons/LoginIcon.js

```diff
@@ -0,0 +1,40 @@
+export const LoginIcon = () => (
+  <svg
```

Comment:
I think that we could optimize this SVG. I'd consider some online tools for this.